app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,39 @@
+<nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white">
+  <a class="navbar-brand ml-5" href="#"><i class='fas fa-dove'></i> SHIZU BnB</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <form class="form-inline my-2 my-lg-0">
+      <input class="form-control mr-sm-2" type="search" placeholder="Anywhere" aria-label="Search">
+      <%= link_to "Search", href="#", class: "btn btn-outline-dark my-2 my-sm-0" %>
+  </form>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav ml-auto">
+    <%= link_to "Become a Host", href="#", class: "btn btn-dark my-2 my-sm-0" %>
+      <li class="nav-item active">
+        <%= link_to "Home", home_pages_home_path, class: "nav-link" %>
```

Comment:
instead of using `home_pages_home_path` you can use the following:
`<%= link_to "Home", root_path, class: "nav-link" %>`